auslib/web/common/rules.py

```diff
@@ -20,6 +20,10 @@ def get_rules():
             where[field] = request.args[field]
 
     rules = dbo.rules.getOrderedRules(where=where)
+    blobs = dbo.releases.getReleaseBlobs(names=[rule['mapping'] or rule['fallbackMapping'] for rule in rules])
+    for rule in rules:
+        rule.update({'has_wnp': blobs[rule['mapping'] or rule['fallbackMapping']]['data'].get('has_wnp', False)})
```

Comment:
This needs a tweak - we should actually be marking `mapping` and `fallbackMapping` separately (see below for more on that).